src/uri.rs

```diff
@@ -0,0 +1,261 @@
+#![allow(dead_code)]
+
+use reqwest;
```

Comment:
Per the build output this import is not used.  Delete.

Reply:
Its currently used within the tests, so i'll move it there.